config/initializers/curation_concerns.rb

```diff
@@ -9,3 +9,13 @@
   end
 end
 
+# We're actually overriding from the sufia-set Sufia::FileSetPresenter, to our
+# own. to_prepare seems necessary to make it stick in dev.
+Rails.application.config.to_prepare do
+  CurationConcerns::MemberPresenterFactory.file_presenter_class = CHF::FileSetPresenter
```

Comment:
To me it makes sense because it brings some consistency to class overrides if we try to do it the same way: class overrides live in `/app/, configuration lives in `config/`. I was only setting a variable when I set it up :) But it's also nice to have it that way already if/when we need to add more overrides to the class in question.